libretroshare/src/jsonapi/jsonapi.h

```diff
@@ -52,6 +52,8 @@ extern JsonApiServer* jsonApiServer;
  */
 struct JsonApiServer : RsSingleJobThread, p3Config
 {
+	static const uint16_t DEFAULT_PORT = 9092 ;
```

Comment:
if you want to put it like that, use `constexpr` instead of `const` ;)

Reply:
done